.github/autobuild/mac.sh

```diff
@@ -146,6 +180,27 @@ pass_artifact_to_job() {
     echo "Moving build artifact to deploy/${artifact}"
     mv ./deploy/Jamulus-*installer-mac.dmg "./deploy/${artifact}"
     echo "artifact_1=${artifact}" >> "$GITHUB_OUTPUT"
+
+    artifact2="jamulus_${JAMULUS_BUILD_VERSION}_mac${ARTIFACT_SUFFIX:-}.pkg"
+    for file in ./deploy/Jamulus_*.pkg; do
+        if [ -f "${file}" ]; then
+            echo "Moving build artifact2 to deploy/${artifact2}"
+            mv "${file}" "./deploy/${artifact2}"
+            echo "artifact_2=${artifact2}" >> "$GITHUB_OUTPUT"
+        fi
+    done
```

Comment:
As per discussion above, suggest avoiding the confusion of a loop with the following change:

```suggestion
    file=(./deploy/Jamulus_*.pkg)
    if [ -f "${file[0]}" ]; then
        echo "Moving build artifact2 to deploy/${artifact2}"
        mv "${file[0]}" "./deploy/${artifact2}"
        echo "artifact_2=${artifact2}" >> "$GITHUB_OUTPUT"
    fi
```